addons/mail/static/src/components/canned_response_suggestion/canned_response_suggestion.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.CannedResponseSuggestion" owl="1">
+        <a class="o_CannedResponseSuggestion dropdown-item" t-att-class="{ 'active': props.isActive }" href="#" role="menuitem" t-att-data-id="cannedResponse.id"
+           t-on-click="_onClick" t-on-mouseover="_onMouseOver">
```

Comment:
no new line between attributes in this case

Reply:
change isn't pushed?

Reply:
It is